### ISSUE TITLE
'start from scratch' sections are not getting revealed

### ISSUE THREAD

**Issue by bass3l** [spring-guides/getting-started-guides#53]
Most of the guides with 'start from scratch' section are not getting revealed when you click on them like 'Build with Maven', I opened the console and found the following error:

`Uncaught TypeError: Cannot read property 'innerHTML' of null`

more specifically on `main.js` line `1867` after prettifying : 
```javascript
       var a = {
                ready: function() {
                    var a = null;
                    e("pre").addClass("prettyprint").each(function(b, c) {
                        a = c.firstChild,
                        a.innerHTML = f.prettyPrintOne(a.innerHTML) // HERE
                    })
                },
                destroy: function() {}
            };
```
using Chrome version 60.0.3112.113

**Comment by dsyer** [comment]
Can you give a specific example? All the guides I looked at have a section called "Build with Maven" and "Build with Gradle", both of which have `|>` icons and hide and reveal features that work for me. None of them has a section called "start from scratch" (just a paragraph with those words in it). The link in that paragraph is always to "Build with Gradle", which seems like a bug, but that's not what you seem to be seeing.

**Comment by bass3l** [comment]
ok, here is one : [rest-service-cors](https://spring.io/guides/gs/rest-service-cors/)

and here's a screenshot for the section that I mean for clarification : 

![image](https://preview.ibb.co/h2K2EQ/Untitled.png)

**Comment by dsyer** [comment]
Right, and that's the same as I see (there is no "start from scratch" section). What is it that doesn't work as expected?

**Comment by bass3l** [comment]
when you click on 'Build with Gradle' for example, no thing happens where it should reveal the section (the accordion) that talks about building with gradle... right?

**Comment by dsyer** [comment]
Works for me. Do you have any errors in the network (404 etc on scripts that didn't download)?

**Comment by dsyer** [comment]
BTW I have the identical version of Chrome, so I would say either your network environment is different (more constrained), or you have Chrome extensions that break this for you.
